include/cadmium/engine/pdevs_dynamic_coordinator.hpp

```diff
@@ -0,0 +1,247 @@
+/**
+ * Copyright (c) 2018, Laouen M. L. Belloli
```

Comment:
It looks like a lot of code is coming from pdevs_coordinator.hpp
Please add me :)

Reply:
Done!